app/routes/missions.einstein-probe/route.mdx

```diff
@@ -33,6 +33,13 @@ The [Einstein Probe (EP)](https://ep.bao.ac.cn/ep/) is a mission of the Chinese
 
 **JSON-Serialized GCN Notices Types in GCN Kafka:**
 
+Einstein Probe distributes alerts and localizations detections of gamma-ray transients. These notices are published on the GCN Kafka topic `gcn.notices.einstein_probe.wxt.alert`.
```

Comment:
The first sentence has a grammatical error.

Reply:
I am following BurstCube template: https://gcn.nasa.gov/missions/burstcube
I tried to rephrase first sentence a bit.
```suggestion
Einstein Probe distributes alerts and localizations for the detection of gamma-ray transients. These notices are published on the GCN Kafka topic `gcn.notices.einstein_probe.wxt.alert`.
```

Reply:
@Vidushi-GitHub, when you receive comments on your PRs, please make the changes yourself --- you needn't suggest the changes first. If you want to use the GitHub PR suggestion feature, that's fine, but please go ahead and apply the suggestion.